notebooks/05_ViT.ipynb

```diff
@@ -52,24 +61,80 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from autocast.data.utils import get_datamodule\n",
+    "if not THE_WELL:\n",
```

Comment:
There's now a `get_datamodule()` function in `data/utils.py` that can be added here.